docs/04-operation-guides/troubleshooting/01-installation-troubleshooting.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Installation troubleshooting
+type: Troubleshooting
+---
+
+## Component doesn't work after successful installation
+
+If the installation is successful but a component does not behave in the expected way, see if all deployed Pods are running. Run this command:
+
+```bash
+kubectl get pods --all-namespaces
+```
+
+The command retrieves all Pods from all Namespaces, the status of the Pods, and their instance numbers. Check if the status is `Running` for all Pods. If any of the Pods that you require do not start successfully, install Kyma again.
```

Comment:
```suggestion
The command retrieves all Pods from all Namespaces, the status of the Pods, and their instance numbers. Check if the status is `Running` for all Pods. If any of the Pods that you require has not started successfully, install Kyma again.
```

I'd use Present Perfect, as you're checking the status of Pods that have already been deployed nad should be running, so you're basically talking about something that should already have happened. Present Simple would be good if you were expected to start the Pods only after reading this.
And I'd use singular, 'cause you only need one to fail for this to be true.